corehq/apps/receiverwrapper/tests/test_submissions.py

```diff
@@ -118,6 +122,91 @@ def test_submit_deprecated_form(self):
         self.assertEqual(self.couch_user.get_id, form.history[0].user)
 
 
+@patch('corehq.apps.receiverwrapper.views.domain_requires_auth', return_value=True)
+class NoAuthSubmissionTest(BaseSubmissionTest):
+    def setUp(self):
+        super(NoAuthSubmissionTest, self).setUp()
+        self.url = self.url + '?authtype=noauth'
```

Comment:
rather just don't auth the client: https://github.com/dimagi/commcare-hq/blob/74adc998673b642f1b8573493ea32af4c6f04355/corehq/apps/receiverwrapper/tests/test_submissions.py#L31

Reply:
👍

Reply:
the param however is necessary to replicate the method calls that actually happen, and the authorisation is not used, will remove it nonetheless

Reply:
ok